pkgs/applications/video/ustreamer/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, stdenv, fetchFromGitHub, libbsd, libevent, libjpeg }:
+
+stdenv.mkDerivation rec {
+  pname = "ustreamer";
+  version = "3.27";
+
+  src = fetchFromGitHub {
+    owner = "pikvm";
+    repo = "ustreamer";
+    rev = "v${version}";
+    sha256 = "1max2171abdpix0wq7mdkji5lvkfzisj166qfgmqkkwc2nh721iw";
+  };
+
+  buildInputs = [ libbsd libevent libjpeg ];
+
+  enableParallelBuilding = true;
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ustreamer $out/bin/
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/pikvm/ustreamer";
+    description = "µStreamer - Lightweight and fast MJPG-HTTP streamer";
+    longDescription = ''
+      µStreamer is a lightweight and very quick server to stream MJPG video from
+      any V4L2 device to the net. All new browsers have native support of this
+      video format, as well as most video players such as mplayer, VLC etc.
+      µStreamer is a part of the Pi-KVM project designed to stream VGA and HDMI
+      screencast hardware data with the highest resolution and FPS possible.
+    '';
+    license = licenses.gpl3;
```

Comment:
gpl3Plus or gpl3Only?

Reply:
that's a good question, i don't know. what would you think after looking at https://github.com/pikvm/ustreamer/blob/master/LICENSE ?

Reply:
ok. sorry, but i still do not understand if this means that it's gpl3plus or -only.

Reply:
i just put `gpl3plus` there because from reading "either version 3 of the License, or (at your option) any later version." i interpret that this means `gpl3plus`